client/lib/posts/utils.js

```diff
@@ -16,11 +16,11 @@ import postNormalizer from 'lib/post-normalizer';
 export const getEditURL = function( post, site ) {
 	let basePath = '';
 
-	if ( ! includes( [ 'post', 'page' ], post.type ) ) {
+	if ( post.type && ! includes( [ 'post', 'page' ], post.type ) ) {
```

Comment:
maybe `const postType = post.type || 'post';` to start, then you don't need the fancy boolean clause or the `|| 'post'` later on?

Reply:
Good shout! Much cleaner.